src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,137 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public static class Node<T> {
+        private T item;
+        private Node<T> prev;
+        private Node<T> next;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
+            this.prev = prev;
+            this.item = value;
+            this.next = next;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (head == null && tail == null) {
+            Node<T> newNod = new Node<>(null, value, null);
+            head = newNod;
+            tail = newNod;
+        } else {
+            Node<T> someNod = new Node<>(tail, value, null);
+            tail.next = someNod;
+            tail = someNod;
+        }
```

Comment:
```suggestion
        Node<T> someNod = new Node<>(tail, value, null);
        if (tail == null) {
            head = someNode;
        } else {
            tail.next = someNod;
        }
        tail = someNod;
```

Reply:
You create the same Node, let's create one Node.
Let's move veriable tail after if.